procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -24,7 +24,6 @@ nearly three weeks, with *three* major days of activity:
 -  Do the prep work a week before the merge
 
    -  `File tracking migration bug <#file-tracking-migration-bug>`__
-   -  `Turn off beta l10n bumper on RC day <#turn-off-beta-l10n-bumper-on-rc-day>`__
```

Comment:
Courtesy to @jcrisau's work in https://bugzil.la/1680028, l10n bumper runs on a daily basis now. Given that, I don't think it's worth doing this operation, which at this point would only save spinning up a no-op (given beta is closed) worker 7 times until next mergeduty Monday.